src/GithubMergeTool/GithubMergeTool.cs

```diff
@@ -303,7 +328,8 @@ git commit
             }
 
             // https://docs.github.com/en/graphql/reference/mutations#enablepullrequestautomerge
-            response = await _client.PostAsyncAsJson("https://api.github.com/graphql", JsonConvert.SerializeObject(new {
+            response = await _client.PostAsyncAsJson("https://api.github.com/graphql", JsonConvert.SerializeObject(new
```

Comment:
nit: formatting change looks unnecessary/inconsistent.

Reply:
Not sure if it is inconsistent.  Other uses of object literal notation in this code file we have a newline before the opening brace.

Reply:
I misread it, sorry 😓